k8s/manifests/kube-prometheus/lib/alphasms.libsonnet

```diff
@@ -171,7 +171,7 @@ local grafanaDashboards = { grafanaDashboards: {
             "useBackend": false
           }
         ],
-        "title": "Balance",
+        "title": "Balance (in $)",
```

Comment:
hope the balance is in USD

Reply:
@roypeter 
As per Arnaud mentioned that the balance can be displayed in any currency, but it is crucial to clearly indicate the currency. This ensures that anyone viewing the metrics in Prometheus can easily distinguish and interpret the values.